nni/contrib/distillation/label_patcher.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from __future__ import annotations
+
+from collections import abc
+import logging
+from os import PathLike
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Literal, Type, cast
+
+from torch.utils.data import DataLoader, Dataset
+
+from .storage import create_storage
+from .uid_dataset import create_uid_dataset, _UidDataset
+from .utils import _default_labels_split_fn, _default_labels_collate_fn
+
+_logger = logging.getLogger(__name__)
+
+
+class DistilLabelPatcher:
```

Comment:
Distill has double-l.

Reply:
distil and distill are both ok, and in distillation, distil seems more normal, i.e. DistilBert, DistilGBT2